templates/app.json.erb

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
How does this template relate to the [manifest generator](https://github.com/thoughtbot/suspenders/blob/5af02dad438ceed099a9b042ad0fc9a0e2feaba0/lib/suspenders/generators/production/manifest_generator.rb#L11)

Reply:
Oh, yikes, I didn't realize this is where this went. I'll update.